### DISCUSSION TITLE
not able to view image of nft in wallet

### DISCUSSION THREAD

**Discussion by sahil-kaushik1** [Cyfrin/foundry-full-course-cu#3086] [Q&A] [answered]
i have made the mood nft and deployed it on anvil 
after importing in the wallet it gets imported but i am not able  to see the images of my nft it shows like this 
![image](https://github.com/user-attachments/assets/5e557909-51f2-4a22-8e2e-1c15ca95ebf1)

nft meta data looks like this 

```
abi.encodePacked(
                    _baseURI(),
                    Base64.encode(
                        bytes(
                            abi.encodePacked(
                                '{"name: "',
                                name(),
                                '", description: "An NFT that reflects your mood!", "attributes": [{"trait_type": "Mood", "value": 100}], "image": ',
                                imageuri,
                                '"}'
                            )
                        )
                    )
                )
```

**Comment by EngrPips** [comment, marked as answer]
Hello @sahil-kaushik1, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2112 and let us know if it helps you resolve the issue.
